tests/Common/ComponentsTest.php

```diff
@@ -2723,6 +2784,41 @@ public function testRevertingConfigRowVersionWillNotCreateEmptyConfiguration()
 
         $originalConfigRow = $originalConfigurationArray['rows'][0];
         $rollbackedConfigRow = $rollbackedConfigurationArray['rows'][0];
-        $this->assertArrayEqualsExceptKeys($originalConfigRow, $rollbackedConfigRow, ['version']);
+        $this->assertSame('Rollback from version 1 (rollbacking config version 2)', $rollbackedConfigRow['changeDescription']);
```

Comment:
Kurňa, tohle je hodně kostrbatá hláška! :-) Už jsem tu měl napsanej elaborát a pak mi došlo, co to vlastně říká.

Jo mimochodem google říká, že spíš `rollback from #{newer} to #{older}`, ale nejsem si jistej, jestli to je použitelný i v tomhle případě?

`Rollback to version 1 (via configuration rollback to version 2)`?

Ty separátní verze řádků a configů jsou dost zmatený :-(

Reply:
Nejvíc zmatený mi na tom přijde, že řádky jsou ~immutable a configy ~mutable a verze se udržují separátně. To byl v tomhle hrozný brainfuck. 

Ale ty tvoje hlášky jsou lepší rozhodně.